src/utilities/constants.js

```diff
@@ -5,3 +5,4 @@ export const APPROVAL_API_BASE = `${process.env.BASE_PATH}/approval/v1`;
 // should be based on locale in future. Dont know why we use en-GB format in mocks
 export const dateOptions = [ 'en-GB', { year: 'numeric', month: 'short', day: 'numeric' }];
 
+export const RBAC_API_BASE = `${process.env.BASE_PATH}/rbac/v1`;
```

Comment:
i think you need to specify whole api version `v1.0`. There were some issues with portfolios and approval 
 requests (adding/editing, fetching workflows etc.) were not executed when just the `v1` path was used.

Reply:
+1

Reply:
`Agree, good to know, I think we will, eventually have to use v1, but will switch to v1.0 for now

Reply:
The approval was changed to v1.0 - the rbac only works with v1, this has to stay as is